python/llm/example/CPU/HF-Transformers-AutoModels/Model/baichuan/README.md

```diff
@@ -8,11 +8,22 @@ To run these examples with IPEX-LLM, we have some recommended requirements for y
 In the example [generate.py](./generate.py), we show a basic use case for a Baichuan model to predict the next N tokens using `generate()` API, with IPEX-LLM INT4 optimizations.
 ### 1. Install
 We suggest using conda to manage environment:
+
+On Linux:
 ```bash
 conda create -n llm python=3.11
 conda activate llm
 
-pip install ipex-llm[all] # install ipex-llm with 'all' option
+pip install --pre --upgrade ipex-llm[all] --extra-index-url https://download.pytorch.org/whl/cpu # install ipex-llm with 'all' option
+pip install transformers_stream_generator  # additional package required for Baichuan-13B-Chat to conduct generation
+```
+
+On Windows:
+```powershell
```

Comment:
As we recommend using cmd for GPU on Windows, maybe it is better with cmd here?
```
```cmd
```

Reply:
Do we need to change line 46 ```powershell to ```cmd to align?
![image](https://github.com/intel-analytics/ipex-llm/assets/109123695/ec9eedf2-3fc9-4c93-94e3-99cb349e154b)

Reply:
I think we can do that :)